src/App.tsx

```diff
@@ -15,48 +16,122 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  visibleGoods.sort((prevGood, nextGood) => {
+    switch (sortType) {
+      case SortType.NONE:
+        return 0;
```

Comment:
if I am right, you don't need this case

Reply:
Why not?

Reply:
OMG! My first solution without any mentor's rejects ))))
Mb, I'll open whiskey this evenig))))